driver/pace/driver/tendency_state.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass, field, fields
+
+from pace import util
+
+
+@dataclass()
+class TendencyState:
```

Comment:
Thanks to your refactors, this is now only used where PhysicsState is also used (since dycore to physics also uses PhysicsState). This makes a lot of sense, since it accumulates physics tendencies to get applied later.

You could do this in a follow-on PR if you want to get this one merged, but could you package this under the physics state? Either as attributes on that state, or as a dataclass attribute? The pace.driver.run module doesn't really need to know this class exists, because it's only needed to implement dycore_to_physics, physics, and physics_to_dycore, but currently there's a fair bit of logic about this class in that file.